generate/seccomp/seccomp_default_linux.go

```diff
@@ -0,0 +1,15 @@
+// +build linux
+
+package seccomp
+
+import "syscall"
```

Comment:
`syscall` is [end-of-life][1]:

> NOTE: This package is locked down. Code outside the standard Go repository should be migrated to use the corresponding package in the golang.org/x/sys repository…

We already vendor `golang.org/x/sys` here; can we use that instead?

[1]: https://golang.org/pkg/syscall/